static/app/views/dashboards/widgetBuilder/widgetLibrary/index.tsx

```diff
@@ -37,6 +37,13 @@ export function WidgetLibrary({
       widget => !(widget.widgetType === WidgetType.RELEASE)
     );
   }
+  if (organization.features.includes('issue-list-better-priority-sort')) {
+    for (const [key, value] of Object.entries(defaultWidgets)) {
+      if (value.id === 'issue-for-review') {
+        defaultWidgets[key].queries.orderby = 'betterPriority';
```

Comment:
@ceorourke this is going to mutate the input btw, we probably should use a fresh copy of `defaultWidgets` for this

Reply:
It doesn't work anyway, I was just pushing it up to show what I attempted